test/system/065-cp.bats

```diff
@@ -7,6 +7,312 @@
 
 load helpers
 
+@test "podman cp file from host to container" {
+    skip_if_remote "podman-remote does not yet handle cp"
+
+    srcdir=$PODMAN_TMPDIR/cp-test-file-host-to-ctr
+    mkdir -p $srcdir
+    # Create multiple files which prevents some (expensive) clean ups via
+    # podman-exec.
```

Comment:
Can you elaborate on this? I didn't understand what you mean by cleanup, or why it's expensive. Do you mean that this helps avoid having to `podman exec rm /path` ?

Reply:
Yes, it was to avoid too many `exec`s. They're expensive and I was worried the tests would consume too much time.